api/tests/Behat/bootstrap/v2/Common/ElementSelectionTrait.php

```diff
@@ -64,6 +64,13 @@ public function iClickOnNthElementBasedOnRegex(string $regex, int $elementIndex)
         $element->click();
     }
 
+    public function clickBasedOnText($text)
+    {
+        $xpath = sprintf('//a[text()[contains(., \'%s\')]]', $text);
+        $link = $this->getSession()->getPage()->find('xpath', $xpath);
+        $link->click();
+    }
+
```

Comment:
Ah missed what this was doing in the original code. There's the built-in mink function that does this in one line:

```
$this->clickLink('Click me!');
```